guide/voice/receiving-audio.md

```diff
@@ -0,0 +1,52 @@
+# Receiving Audio
+
+::: warning
+Discord does not officially support Discord Bots receiving audio ([see here](https://github.com/discordapp/discord-api-docs/issues/808)), so there may be some breakages. However, Discord.js aims to reasonably support this feature.
+:::
+
+## Basic Usage
+
+In addition to sending audio over voice connections, you can also receive audio (i.e. listen to other users and bots in a voice channel) using Discord.js.
+
+The example below will listen to a user until they stop speaking, and all the audio received from that user is decoded from Opus to signed 16-bit little-endian (s16le) PCM and stored in a file called `user_audio`.
+
+```js
+const fs = require('fs');
+
+// Create a ReadableStream of s16le PCM audio
+const audio = connection.receiver.createStream(user, { mode: 'pcm' });
+
+audio.pipe(fs.createWriteStream('user_audio'));
+```
+
+The `mode` option defaults to `'opus'` and can be specified as `'pcm'`. When `'opus'` is specified, Discord.js will not attempt to decode each received Opus packet. The stream created in this case will be a ReadableStream of Opus packets.
+
+With `'pcm'` specified, Discord.js will attempt to decode each received Opus packet into PCM, giving you a stream of raw audio that can be processed by other applications.
+
+Additionally, the `end` option can be specified. It defaults to `'silence'`, which ends the ReadableStream once the user stops talking. It can be set to `'manual'`, which means the stream doesn't end until you end it yourself. Note that Discord.js will not interpolate silence into gaps in the audio (where the user has stopped speaking), this is something you will have to do yourself.
+
+::: tip
+To work with PCM audio, you could use software such as [Audacity.](https://www.audacityteam.org/) To import the audio into Audacity, **File > Import > Raw Data...** and then select your audio file. You should select **Signed 16-bit PCM** as the encoding, a **Little-endian** byte order, **2 Channels (Stereo)** and a sample rate of **48000Hz**. 
```

Comment:
Should put an "open" before the Audacity interface instructions, i.e. "open **File > Import > Raw Data...**", otherwise it reads a bit awkwardly.